packages/pn-commons/src/components/SessionModal.tsx

```diff
@@ -36,12 +36,11 @@ const SessionModal = ({
       <DialogTitle sx={{ textAlign: 'center' }}>{title}</DialogTitle>
```

Comment:
Rivedere in generale lo style e invece di usare i pixel, usare le formule del tipo py, pt, pb, px = valore numerico (1 = 8px, 2 = 16px, 3 = 24px)